docs/README.md

```diff
@@ -2059,6 +2059,26 @@ the current timestamps. Query cache can be enabled after the backfilling is comp
 An alternative solution is to query `/internal/resetRollupResultCache` url after backfilling is complete. This will reset
 the query cache, which could contain incomplete data cached during the backfilling.
 
+Use the next [curl](https://curl.se/) commands to reset the query cache on [VictoriaMetrics Single](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html):
```

Comment:
lets move this example to https://docs.victoriametrics.com/url-examples.html
and link it here